pytorch-yolo-image/README.md

```diff
@@ -0,0 +1,32 @@
+# YOLOV8 detection Example For WASI-NN with PyTorch Backend
+
+Code for a working example of object detection using the yolov8 model. 
+
+To build (from the `/rust` directory):  
+`cargo build --release --target wasm32-wasi`
+
+To compile the WASM AOT (Ahead of Time) - this results in a much more performant binary  
+```bash
+wasmedge compile ./target/wasm32-wasi/release/wasmedge-wasinn-example-yolo-image.wasm ./target/wasm32-wasi/release/wasmedge-wasinn-example-yolo-image.wasm
+```
+
+
+To Run (from root `/` directory of the example):  
+```bash
+wasmedge --dir .:. ./rust/target/wasm32-wasi/release/wasmedge-wasinn-example-yolo-image.wasm ./yolov8n.torchscript ./dog.png
```

Comment:
This should be `wasmedge --dir .:. ./rust/target/wasm32-wasi/release/wasmedge-wasinn-example-yolo-image.wasm ./yolov8n.torchscript ./input.jpg` since dog.png doesn't exist.

Reply:
Thanks, missed this while cleaning up.